nodejs/docs/node-standard-and-best-practices.md

```diff
@@ -417,24 +424,144 @@ Returns an array with all the values from the original array for which the funct
    // filtered is [12, 130, 44]
 ```
 
-### 6.5- Others
+### 6.5- Some
+
+Returns true when at least one array's element evaluated by the function, returns a truthy value.
+
+```javascript
+   const isLegal = value => value >= 18;
+
+   const anyCanDrive = [8, 37, 17, 62, 13].some(isLegal);
+   // returned true because the elements 37 and 13 are greater than 18
+
+```
+
+
+### 6.6- Every
+
+Returns true when all the elements within the array evaluated by the function, return a truthy value.
+
+```javascript
+   const  isEven = value => !(value % 2);
+
+   const  areEven = [8, 5, 17, 40, 13].every(isEven);
+   // returned false because the elements 5,17 and 13 are odd
+
+```
 
-Other useful methods for a _declarative_ approach.
 
-* **some()**.
-* **every()**.
-* **forEach()**.
-* **split()**.
+
+### 6.7- forEach
+
+Execute the function for each element without returning a value. The forEach function doesn't modify the original array.
+
+```javascript
+   const tennisPlayers = ['Federer','Nalbandian','Nadal'];
+
+   tennisPlayers.forEach(name => {
+     //code that is will executed for each element
```

Comment:
I think is very important to mention that with forEach you create a new scope!
Many times I have seen errors declaring vars here which can not be used after the forEach. This is caused because in other languages the forEach doesnt create a new scope, so people dont notice it